group_vars/all.yml

```diff
@@ -1,3 +1,4 @@
 ---
 operator_namespace: '{{ meta.namespace }}'
 clustername: 'myk8scluster'
+kernel_cache_drop_svc_port: 9222
```

Comment:
Can this be a default in the fio role instead of being hard coded here? Also, is this going to be user configurable or is it always going to be port 9222

Reply:
the intent was to to make kernel cache dropper a feature that could be used by multiple benchmarks, but that's not the way it's implemented today.   So you mean it could be in roles/fio_distributed/defaults/main.yml or something like that, at least for now?

Reply:
Would each benchmark be launching its own kernel cache dropper pods?

Reply:
Wondering if it makes sense to have it managed outside the individual benchmarks. Might make sense to treat it kind of like the backpack daemonset option that sits above the launching of the benchmark. So we could check if a var is set and then launch the specified pods. That said I'm guessing we need to know where the fio/other benchmark pods are going to land.

Reply:
@dry923 I like the overall idea.   Specifically I don't like having to put code in the fio benchmark specifically for cache dropping (e.g. finding the pod IPs, formatting them into a list, putting them in a configmap, getting them out of the configmap, sigh).   I did not understand how to pull this out of fio benchmark and make it benchmark-independent, would it be a separate role that fio benchmark would invoke?

Reply:
So It now is managed outside the individual benchmarks by a separate role.   Much simpler.